Runtime/Scripts/SceneImporter/ImporterMaterials.cs

```diff
@@ -706,6 +716,9 @@ void SetTransformKeyword()
 			// {
 			// 	MaterialExtensions.ValidateMaterialKeywords(pbrGraphMap.Material);
 			// }
+			
+			HDMaterial.ValidateMaterial(mapper.Material);
+			HDMaterial.ValidateMaterial(vertColorMapper.Material);
```

Comment:
Please add a conditional check so that we dont break the plugin for someone not using HDRP.

How to do it is outlined here:
https://discussions.unity.com/t/check-if-hdrp-is-present-in-project-c/799532